src/hooks/directoryHooks/useGetSecondLevelFoldersAndPages.ts

```diff
@@ -0,0 +1,45 @@
+import { useQueries, UseQueryResult } from "react-query"
+
+import { DIR_SECOND_LEVEL_CONTENT_KEY } from "constants/queryKeys"
+
+import * as DirectoryService from "services/DirectoryService/index"
+
+import {
+  DirectoryData,
+  PageData,
+  SecondLevelFoldersAndPagesParams,
+} from "types/directory"
+import { isDirectoryData } from "utils"
+
+interface CollectionOrder {
+  collectionName: string
+  order: string[]
+}
+
+const getCollectionOrder = (
```

Comment:
This API should take the data directly and return the collection order instead of wrapping over directory service.

This is because the method is not concerned with the fetching of data, but the view of said data

Reply:
Merged this function into the parent hook in 1e8b8b84f3c2aa810746aea11ae3077bb6bce490.